### PR TITLE
Adding Jackal's v4 Testnet

### PR DESCRIPTION
**Pull request by TheMarstonConnell**
Adding Jackal's secondary testnet with archway IBC routes.

### CODE REVIEW

File: testnets/jackaltestnet2/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "display": "jkl",
+      "symbol": "JKL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.svg"
+      },
+      "coingecko_id": "jackal",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.png",
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/assetlist.json:28]: Please remove the duplicate uploads, and use image sync instead (they already exist at ~/jackal/images/).

File: testnets/jackaltestnet2/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        {
+          "denom": "jkl",
+          "exponent": 6
+        }
+      ],
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
+      "logo_URIs": {
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/assetlist.json:21]: Please omit the `logo_URIs` property

File: testnets/jackaltestnet2/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        },
+        {
+          "denom": "jkl",
+          "exponent": 6
+        }
+      ],
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/assetlist.json:20]: please define a `trace` to mainnet jackal using a trace "type": "test-mintage"

File: testnets/jackaltestnet2/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "jackaltestnet",
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/assetlist.json:3]: the chain_name must match the directory name (jackaltestnet2)

File: testnets/jackaltestnet2/chain.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "jackal-v4-testnet",
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/chain.json:3]: chain_name must match directory name (jackaltestnet2)

File: testnets/jackaltestnet2/chain.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  "chain_id": "mesomelas-1",
+  "website": "https://jackalprotocol.com",
+  "pretty_name": "Jackal v4",
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/chain.json:6]: What is happening with the first jackaltestnet?

File: testnets/jackaltestnet2/assetlist.json (hunk 3 above)
Comment (TheMarstonConnell) [reply testnets/jackaltestnet2/assetlist.json:20]: I've never seen this before in the asset lists, do you have an example of another chain using this field?

Comment (JeremyParish69) [reply testnets/jackaltestnet2/assetlist.json:20]: E.g., KYVE Testnet uses this correctly
https://github.com/cosmos/chain-registry/blob/78d62856452d99890e2486babd0c1abd831da934/testnets/kyvetestnet/assetlist.json#L23

File: testnets/jackaltestnet2/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
+      "coingecko_id": "jackal",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/jackal/images/jkl.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/jackal/images/jkl.svg"
+        }
```
Comment (JeremyParish69) [inline testnets/jackaltestnet2/assetlist.json:26]: You can also see how it has image_sync (although I will eventually be generated automatically once the trace is defined)

**Pull request merged**